libs/prisma-models/src/selected_fields.rs

```diff
@@ -11,6 +11,25 @@ pub struct SelectedFields {
     pub scalar: Vec<SelectedScalarField>,
     pub relation: Vec<SelectedRelationField>,
 }
+impl SelectedFields {
+    pub fn only_scalar_and_inlined(&self) -> SelectedFields {
```

Comment:
In quite a few cases we passed in relation fields that are completely virtual. The result was that queries then tried to query columns such as `Todo.tags`